mono_repo/lib/src/commands/pub.dart

```diff
@@ -131,5 +131,12 @@ final String _sdkDir = (() {
   return aboveExecutable;
 })();
 
-final String pubPath =
+final String _pubPath =
     p.join(_sdkDir, 'bin', Platform.isWindows ? 'pub.bat' : 'pub');
+
+/// The path to the root directory of the Flutter SDK.
+/// Requires the "FLUTTER_HOME" environment variable to be set.
+final String _flutterHome = Platform.environment['FLUTTER_HOME'] ?? '';
+
+final String _flutterPath =
```

Comment:
I'd change this.

If `FLUTTER_HOME` exists, use `$FLUTTER_HOME/bin/flutter[.bat]` otherwise use `flutter[.bat]`.

Right?

Reply:
Okay, I've changed it a bit, can you check? Otherwise, feel free to ammend as you see fit.

Reply:
@kevmoo have you had a chance to look at the updated code yet?